sdl_android/src/main/java/com/smartdevicelink/protocol/WiProProtocol.java

```diff
@@ -678,7 +753,23 @@ else if (frameInfo == FrameDataControlFrameType.StartSession.getValue()) {
 					int serviceDataAckSize = BitConverter.intFromByteArray(packet.getPayload(), 0);
 					handleProtocolServiceDataACK(serviceType, serviceDataAckSize,(byte)packet.getSessionId ());
 				}
-			} //TODO else if register transport frame
+			} else if (frameInfo == FrameDataControlFrameType.RegisterSecondaryTransportACK.getValue()) {
+				handleRegisterSecondaryTransportACK((byte) packet.sessionId);

```

Comment:
Only circular logic exists from this call. WiProtocol -> IProtocolListener -> WiProProtocol